Automattic/wp-calypso
# Coming Soon: check to see if the ETK plugin is installed and hide the setting if so

## Changes proposed in this Pull Request

Hi friends!

Because the new coming soon functionality lives in the [WordPress.com Editing Toolkit](https://github.com/Automattic/wp-calypso/tree/master/apps/editing-toolkit/editing-toolkit-plugin), if an Atomic users wishes to deactivate or delete that plugin we don't wish to allow them to toggle Coming Soon in the WPCOM settings (because it won't have any effect!)

So we're going to hide the coming soon option when the ETK plugin is disabled/deleted.

## Testing instructions

Create an Atomic site by upgrading to Business/Ecommerce

Check that the WordPress.com Editing Toolkit plugin is installed and activated at {your_site}/wp-admin/plugins.php?plugin_status=all&paged=1&s

Head over to http://calypso.localhost:3000/settings/general/{your_site}?flags=coming-soon-v2

You should be able to toggle Coming Soon mode

<img width="745" alt="Screen Shot 2020-11-19 at 2 36 03 pm" src="https://user-images.githubusercontent.com/6458278/99620730-334d1b00-2a7a-11eb-8b95-a9f81eb176e2.png">

The badge should be visible when Coming Soon mode is on

<img width="338" alt="Screen Shot 2020-11-19 at 3 18 11 pm" src="https://user-images.githubusercontent.com/6458278/99620843-74452f80-2a7a-11eb-868c-e8afd9ecf4cd.png">

Now deactivate the plugin

<img width="939" alt="Screen Shot 2020-11-19 at 2 36 09 pm" src="https://user-images.githubusercontent.com/6458278/99620759-452ebe00-2a7a-11eb-896b-fb9cd6c90930.png">

Return to the settings page: http://calypso.localhost:3000/settings/general/{your_site}?flags=coming-soon-v2

You should see the following

<img width="721" alt="Screen Shot 2020-11-19 at 7 20 44 pm" src="https://user-images.githubusercontent.com/6458278/99639988-71f3cd00-2a9c-11eb-8242-d3f043216817.png">

And the badge should be gone

<img width="296" alt="Screen Shot 2020-11-19 at 3 17 39 pm" src="https://user-images.githubusercontent.com/6458278/99620854-7dce9780-2a7a-11eb-8216-e2d3031370ef.png">


Switch to a simple site

Ensure that Coming Soon mode is working and you can toggle it on and off.
